app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
True, the idea of MVP is that presenter is platform independent, but this does not mean that it have to be placed in separate package. If we would need to make it multi-platform then placement in separate package won't change anything. We would have to be place them in separate modules. Packages can be the same. I feel that this discussion ideological, and I prefer pragmatic thinking. For me it is just harder to work on view when I have presenter and view on separate packages. It is also harder to read project when I enter github and I have to search for each element in separate place. Unless there are some reasons I don't see?